clients/client-python/gravitino/exceptions/base.py

```diff
@@ -54,6 +54,9 @@ class InternalError(GravitinoRuntimeException):
 class NotFoundException(GravitinoRuntimeException):
     """Base class for all exceptions thrown when a resource is not found."""
 
+    def __init__(self, message, *args):
```

Comment:
I think the `__init__` function here is not necessary, because the parent class (`GravitinoRuntimeException` here) has implemented, except we want some custom features for this class only.

Reply:
Removed!